hack/sss.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- name: IMAGE_TAG
+  required: true
+- name: IMAGE_NAME
+  value: quay.io/app-sre/managed-release-bundle
+- name: OPENSHIFT_MAJOR_MINOR_VERSION
+  value: "4.16"
+metadata:
+  name: selectorsyncset-template
+objects:
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    name: release-bundle-config-${OPENSHIFT_MAJOR_MINOR_VERSION}
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+      matchExpressions:
+        - key: hive.openshift.io/version-major-minor
+          operator: In
+          values:
+            - "${OPENSHIFT_MAJOR_MINOR_VERSION}"
+    resourceApplyMode: Sync
+    enableResourceTemplates: true
+    resources:
+    - apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: osd-cluster-metadata
+        namespace: openshift-config
+      data:
+        api_openshift_com_name: "{{ fromCDLabel \"api.openshift.com/name\" }}"
+        api_openshift_com_product: "{{ fromCDLabel \"api.openshift.com/product\" }}"
+        api_openshift_com_fedramp: "{{ fromCDLabel \"api.openshift.com/fedramp\" }}"
+        api_openshift_com_sts: "{{ fromCDLabel \"api.openshift.com/sts\" }}"
+        api_openshift_com_limited-support: "{{ fromCDLabel \"api.openshift.com/limited-support\" }}"
+        api_openshift_com_private-link: "{{ fromCDLabel \"api.openshift.com/private-link\" }}"
+        api_openshift_com_ccs: "{{ fromCDLabel \"api.openshift.com/ccs\" }}"
+        api_openshift_com_channel-group: "{{ fromCDLabel \"api.openshift.com/channel-group\" }}"
+        api_openshift_com_environment: "{{ fromCDLabel \"api.openshift.com/environment\" }}"
+        hive_openshift_io_cluster-platform: "{{ fromCDLabel \"hive.openshift.io/cluster-platform\" }}"
+        hive_openshift_io_cluster-region: "{{ fromCDLabel \"hive.openshift.io/cluster-region\" }}"
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
```

Comment:
any specific reason for 2 SSSs?

Reply:
We broke them apart as a part of trying to figure out how to test pre-release bundles, i.e. provision a cluster that doesn't have the release bundle on it but we need the configmap to template the bundle that _we_ install. I haven't figured out the labels we discussed yet here https://redhat-internal.slack.com/archives/C06HVJHJ1H9/p1713449926386489

Reply:
oh got it, thanks!